### DISCUSSION TITLE
v0.4.1

### DISCUSSION THREAD

**Discussion by carson-katri** [carson-katri/dream-textures#818] [Announcements]
This release contains fixes for the render pass, model search/download, and ROCm dependencies.

## Choose Your Installation
Separate versions are available depending on your GPU and Blender version.

Click on the link to download the correct version:

| | Blender 4.1+ | Blender 3.6 → 4.0 |
| -------- | ------------ | ------------------------- |
| **Windows** | | |
| NVIDIA | [dream_textures-windows-cuda-4-1.7z](https://github.com/carson-katri/dream-textures/releases/download/0.4.1/dream_textures-windows-cuda-4-1.7z) | [dream_textures-windows-cuda.7z](https://github.com/carson-katri/dream-textures/releases/download/0.4.1/dream_textures-windows-cuda.7z) |
| AMD/Other | [dream_textures-windows-directml-4-1.zip](https://github.com/carson-katri/dream-textures/releases/download/0.4.1/dream_textures-windows-directml-4-1.zip) | [dream_textures-windows-directml.zip](https://github.com/carson-katri/dream-textures/releases/download/0.4.1/dream_textures-windows-directml.zip) |
| | | |
| **macOS** | | |
| Apple Silicon (M1+) | [dream_textures-macos-arm-4-1.zip](https://github.com/carson-katri/dream-textures/releases/download/0.4.1/dream_textures-macos-arm-4-1.zip) | [dream_textures-macos-arm.zip](https://github.com/carson-katri/dream-textures/releases/download/0.4.1/dream_textures-macos-arm.zip) |
| | | |
| **Linux** | | |
| Any GPU | [Install Manually](https://github.com/carson-katri/dream-textures/wiki/Setting-Up-a-Development-Environment) | [Install Manually](https://github.com/carson-katri/dream-textures/wiki/Setting-Up-a-Development-Environment) |

> [!IMPORTANT]
> ### Windows Installation Instructions
> Due to file size limits on GitHub, the add-on ZIP file is contained in a 7-Zip archive.
> 1. Install [7-Zip](https://www.7-zip.org/)
> 2. Right-click on the downloaded `dream_textures-windows-cuda-*.7z` archive
> 3. Select "7-Zip" > "Extract Here" to get the `dream_textures-windows-cuda-*.zip` file.
> 4. In Blender preferences open "Add-ons" > "Install..."
> 5. Choose the `dream_textures-windows-cuda-4-1.zip` file.

## Setup
After installing the appropriate add-on ZIP file, enable the add-on and expand its preferences. There you will find further instructions for setup.

See the [setup guide](https://github.com/carson-katri/dream-textures/wiki/Setup) for more detailed instructions.

## What's Changed
* Fix Blender 4.1 Render Pass by @NullSenseStudio in https://github.com/carson-katri/dream-textures/pull/808
* Update Hugging Face utilities by @Wauplin in https://github.com/carson-katri/dream-textures/pull/813
* Remove `nightly` prefix from ROCm `extra-index-url` by @carson-katri in https://github.com/carson-katri/dream-textures/pull/811
* Release v0.4.1 by @carson-katri in https://github.com/carson-katri/dream-textures/pull/816

## New Contributors
* @Wauplin made their first contribution in https://github.com/carson-katri/dream-textures/pull/813

**Full Changelog**: https://github.com/carson-katri/dream-textures/compare/0.4.0...0.4.1

<hr /><em>This discussion was created from the release <a href='https://github.com/carson-katri/dream-textures/releases/tag/0.4.1'>v0.4.1</a>.</em>

**Comment by neotmhy** [comment]
Has support for Intel Macs been dropped? I have a 16-Inch, 2019 MacBook Pro with a 2.3 GH 8-Core Intel Core i9 CPU and an AMD Radeon Pro 5500M 8 GB GPU, and 64 GB 2667 MHz DDR4.

**Comment by glandularzorro-hub** [comment]
Tried using it but it fails to finish downloading the dependencies.

**Reply by glandularzorro-hub** [reply]
Ive tried re downloading it and re enabling it but to no avail
